optionals/arcadian/script_component.hpp

```diff
@@ -0,0 +1,6 @@
+#define COMPONENT gt armory arcadian
```

Comment:
This can't have spaces.
```suggestion
#define COMPONENT arcadian
```